graphdatascience/tests/integration/test_ogb_loader.py

```diff
@@ -55,7 +53,24 @@ def test_graph_load_ogbn_mag(gds: GraphDataScience) -> None:
     G.drop()
 
 
-@pytest.mark.enterprise
+@pytest.mark.ogb
+@pytest.mark.compatible_with(min_inclusive=ServerVersion(2, 1, 0))
+def test_graph_load_ogbl_wikikg2(gds: GraphDataScience) -> None:
```

Comment:
There are several more asserts I would like to have in this test (see `test_graph_load_ogbn_arxiv above`)